deeprvat/deeprvat/config.py

```diff
@@ -111,7 +333,9 @@ def update_config(
             else:
                 logger.info("Not performing EAC filtering of baseline results")
             logger.info(f"  Correcting p-values using {correction_method} method")
-            alpha = config.get("alpha_seed_genes", config.get("alpha"))
```

Comment:
I think `alpha` doesn't exist any more at the top level of the config, so `config.get("alpha")` won't work. I think it must be `config["evaluation"]["alpha"]`